scripts/apps/monitoring/controllers/AggregateCtrl.ts

```diff
@@ -53,10 +52,31 @@ export function AggregateCtrl($scope, desks, workspaces, preferencesService, sto
     };
     this.activeFilterTags = {};
 
+    Promise.all(
+        Object.values(extensions)
+            .map(
+                (extension) =>
+                    extension.activationResult?.contributions?.personalSpace?.getSections?.(),
```

Comment:
You probably need a default value here to avoid `undefined` -  `.getSections?.() ?? []`